cogs/make_member.py

```diff
@@ -153,16 +151,21 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
 
         guild_member_ids: set[str] = set()
 
-        request_headers: dict[str, str] = {
+        REQUEST_HEADERS: dict[str, str] = {
             "Cache-Control": "no-cache",
             "Pragma": "no-cache",
             "Expires": "0",
         }
-        request_cookies: dict[str, str] = {
+        REQUEST_COOKIES: dict[str, str] = {
```

Comment:
Unnecessary capitalisation. The request cookies should be allowed to be edited between initial declaration and usage.